site/en/guide/intro_to_graphs.ipynb

```diff
@@ -14,7 +14,7 @@
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "cellView": "both",
```

Comment:
Is colab doing this?

I think I have seen this in every PR where you nbfmt. Maybe wanna add a copybara transform so that this doesn't happen everytime.

Reply:
Wollf didn't fix it when I pointed it out in his CL 🤷‍♂️

Reply:
Maybe add a copybara transform so that this doesn't happen again?

Reply:
But for automation, if there's a way to add nbfmt to the export then it would handle this

Reply:
The automation is copybara. I don't know if its possible to run a script but for now, let's add this to copybara so that the diff churn is minimum.

Reply:
I think that's ot-of-scope for this :) I"m just running nbfmt
ANd I'd rather not rebuild nbfmt in the form of copybara transforms

Reply:
Yeah.. I am not saying to add the copybara transform here but in a separate CL (not PR).

> nbfmt in the form of copybara transforms

I am not asking you to do that. The purpose of nbfmt is to reduce diff churn. For cell-view that's not the case. There are many similar copybara transform running of `<` and `>`, so adding this shouldn't be much of an issue. Also, majority of the recents nbfmt PR's consist of the cell-view transformation.

Reply:
> majority of the recents nbfmt PR

I don't believe this is the case. you're just noticing this in the same notebook that was called out and not fixed. please manage the copybara config how you like with @MarkDaoust , it's added to nbfmt to catch issues that slip by (like it did).

Reply:
> Maybe add a copybara transform so that this doesn't happen again?

Colab's just saving the view state. Adding it to copybara is a bit trickier because we want all transforms to be reversible, but this isn't something you'd want to add back. There's probably a way to add come non-reversible transforms but that's more complicated.

If you want to enforce nbfmt on CLs you could add a pytest.